addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,351 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rfxcom"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="blind1">
+        <label>RFXCOM Blind1 thing</label>
+        <description></description>
+
+        <channels>
+            <channel id="command" typeId="command-channel"/>
+            <channel id="shutter" typeId="shutter-channel"/>
+            <channel id="signalLevel" typeId="signallevel-channel"/>
+            <channel id="batteryLevel" typeId="batterylevel-channel"/>
+        </channels>
+    </thing-type>
+
+    <thing-type id="curtain1">
+        <label>RFXCOM Curtain1 thing</label>
+        <description></description>
+
+        <channels>
+            <channel id="command" typeId="command-channel"/>
+            <channel id="shutter" typeId="shutter-channel"/>
+            <channel id="signalLevel" typeId="signallevel-channel"/>
+            <channel id="batteryLevel" typeId="batterylevel-channel"/>
+        </channels>
+    </thing-type>
+
+    <thing-type id="energy">
+        <label>RFXCOM Energy thing</label>
```

Comment:
Maybe "RFXCOM Energy Sensor", "RFXCOM Humidity Sensor", "RFXCOM Wind Sensor" etc. would be better?